stories/components/toolbar.stories.tsx

```diff
@@ -559,3 +563,117 @@ export function TooltipHelpContentStory({ intent }: ToolbarProps) {
 }
 
 TooltipHelpContentStory.storyName = 'TooltipHelpContent';
+
+const PlaceHolder = () => (
+  <div
+    style={{
+      height: '100%',
+      width: '100%',
+      backgroundColor: 'rgba(147, 197, 253)',
+      display: 'flex',
+      justifyContent: 'center',
+      alignItems: 'center',
+      fontSize: 24,
+    }}
+  >
+    Place Holder
+  </div>
+);
+export function ActivityToolbar() {
+  const [selected, setSelected] = useState<string[]>([
+    'phone',
+    'add-column-left',
+  ]);
+
+  return (
+    <div
+      style={{
+        display: 'flex',
+        height: '100%',
+      }}
+    >
+      {selected.length > 0 ? (
+        <SplitPane size="30%" controlledSide="end">
+          <PlaceHolder />
+
+          <div
+            style={{
+              flexGrow: 1,
+              display: 'flex',
+              flexDirection: 'column',
+              gap: '5px',
+            }}
+          >
+            {itemsBlueprintIcons
+              .filter(({ id }) => selected.includes(id))
+              .map(({ id }) => (
+                <Card
```

Comment:
Not convinced with how it currently looks with the card.

Maybe use an elevation of 0?

Reply:
You can check the diffrance